src/main/webapp/index.html

```diff
@@ -4,8 +4,20 @@
     <meta charset="UTF-8">
     <title>Student Recommendations</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/styles/fonts.css">
+    <link rel="stylesheet" href="/styles/common.css">
   </head>
   <body>
-    <h1>Welcome to Rania and Lorna's capstone project</h1>
+    <div class="header">
+      <h1>StudentRecs</h1>
+      <h3>A recommendation system for and by students!</h3>
```

Comment:
The wording we discussed might be a bit better here - "a recommendation system for students, by students!".

Reply:
thank you for the suggestions Lorna! :D I have just changed it accordingly